test/Microsoft.Azure.Devices.Edge.Test.Common/Platform.cs

```diff
@@ -1,12 +1,14 @@
 // Copyright (c) Microsoft. All rights reserved.
 namespace Microsoft.Azure.Devices.Edge.Test.Common
 {
+    using System;
     using System.Runtime.InteropServices;
+    using Microsoft.Azure.Devices.Edge.Util;
 
     public static class Platform
     {
-        public static IEdgeDaemon CreateEdgeDaemon(string installerPath) => IsWindows()
-            ? new Windows.EdgeDaemon(installerPath)
+        public static IEdgeDaemon CreateEdgeDaemon(Option<string> installerPath) => IsWindows()
```

Comment:
should we keep string type and use Precondition methods to validate it?

Reply:
Would that be better than `Option<string>`?